docs/tutorials/index/index.md

```diff
@@ -42,6 +42,16 @@ General tutorials to help feed your curiosity.
         <td>Demonstrating token transfer and Uniq purchasing</td>
         <td><a href="../fundamentals/tutorial-token-transfer-and-nft-purchase">Link</a></td>
     </tr>
+    <tr>
+        <td>Tutorial - Purcashing UOS token</td>
+        <td>Tutorial for purchasing native UOS token</td>
+        <td><a href="../fundamentals/tutorial-buy-uos">Link</a></td>
+    </tr>
+    <tr>
+        <td>Tutorial - Swapping UOS token</td>
+        <td>Tutorial for swapping UOS tokens.</td>
```

Comment:
Also, move to be a guide and add to the description that it is swap UOS to Ethereum